src/decoder/mod.rs

```diff
@@ -30,29 +36,36 @@ impl<R> Decoder<R>
     /// Builds a new decoder.
     ///
     /// Attempts to automatically detect the format of the source of data.
-    pub fn new(data: R) -> Result<Decoder<R>, DecoderError> {
-        let data = match wav::WavDecoder::new(data) {
-            Err(data) => data,
+    pub fn new(_data: R) -> Result<Decoder<R>, DecoderError> {
+        #[cfg(feature = "wav")]
```

Comment:
I'd prefer if the variable names didn't have a _ prefix, and you just put a `#![allow(unused_variables)]` in the function instead.